### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by hwbllmnn**
Adds a few tests and travis, coveralls and greenkeeper.

@terrestris/devs Please review.

### CODE REVIEW

**Comment by hwbllmnn** [comment]
Please note that the failing test will pass once #6 is merged.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
 dist
+coverage
```
Comment (jansule) [inline .gitignore]: Minor: Please add newline

File: src/ChartRenderer/ChartRenderer.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import select from 'd3-selection/src/select';
+// import {select} from 'd3-selection';
```
Comment (jansule) [inline src/ChartRenderer/ChartRenderer.js]: Minor: Please remove commented code.

Comment (hwbllmnn) [reply src/ChartRenderer/ChartRenderer.js]: Fixed in 1848c85

File: .gitignore (hunk 1 above)
Comment (hwbllmnn) [reply .gitignore]: Fixed in 1848c85

**Pull request merged**